test/e2e/framework.go

```diff
@@ -197,6 +204,12 @@ type TestOptions struct {
 	windowsVMs          string
 }
 
+type tlsTestOptions struct {
```

Comment:
this is not great: the name is too generic and not very accurate (nothing is really related to TLS). This is only useful for flow visibility tests, so why not call it `flowVisibilityTestOptions`?

Reply:
Fixed. Thanks.